mpas_analysis/shared/mpas_xarray/mpas_xarray.py

```diff
@@ -363,7 +423,6 @@ def remove_repeated_time_index(ds):  # {{{
     return ds  # }}}
 
 
-
 def test_load_mpas_xarray_datasets(path):  # {{{
```

Comment:
Yes, I think we should move them to `test`.  But again I think this is a tangent from the kind of feedback I'm interested in getting on this PR at this point.